sheet_to_triples/xl.py

```diff
@@ -39,8 +39,13 @@ def as_rows(row_iter, required_headers):
 
 
 def advance_headers(row_iter, required_headers):
+    best_match = set()
     for row in row_iter:
         values = [cell.value for cell in row]
         if required_headers.issubset(values):
             return values
-    raise ValueError('required headers not found')
+        headers_present = set(values).intersection(required_headers)
```

Comment:
Very slightly better to spell this as:
```python
headers_present = required_headers.intersection(values)
```
Could also use the same result instead of the `issubset()` check, but that's probably not any clearer.